src/e2e-test/features/source.control.management.sync.apps.feature

```diff
@@ -29,6 +29,44 @@ Feature: Source Control Management - Pulling and pushing applications
     Then Banner is shown with message "Successfully pushed pipeline test_pipeline2_fll_airport"
     Then Clean up pipeline "test_pipeline2_fll_airport" which is created for testing
 
+  @SOURCE_CONTROL_MANAGEMENT_TEST
+  Scenario: Should successfully push and pull multiple pipelines to git from sync page
+    # Setup
+    When Deploy and test pipeline "test_multi_push_fll_airport" with pipeline JSON file "fll_airport_pipeline2.json"
+    And Deploy and test pipeline "test_multi_push_logs_generator" with pipeline JSON file "logs_generator.json"
+    # Push 2 pipelines in bulk
+    And Open Source Control Sync Page
+    And Select local pipeline "test_multi_push_fll_airport"
+    And Select local pipeline "test_multi_push_logs_generator"
+    And Push selected pipelines to remote with commit message "upload pipelines to git"
+    And Verify "push" operation is running
```

Comment:
we should move to another page and back to the list. validation initial data load